### PR TITLE
[# 112391353] As a PM/Eng manager so I know who to nudge for acceptance, show me the number of hours a story has been awaiting acceptance

### PR DESCRIPTION
**Pull request by kquet**
[Tracker #112391353](https://www.pivotaltracker.com/story/show/112391353)

### CODE REVIEW

File: lib/app/views/delivered.slim [hunk 1]
```diff
@@ -8,7 +8,8 @@ h2.text-center.short Delivered Stories
         th Story Name
         th Requestor
         th Owners
-        th Age (days)
+        th Age
```
Comment (soonahn) [inline lib/app/views/delivered.slim]: Where are the locales bro?  What if I eventually want to read the site in Korean or French?  hu? (lol jk)

**Comment by kenberland** [comment]
Seriously @soonahn ?

**Comment by soonahn** [comment]
No, I was joking, hence the jk.  I told Kaël that I was joking afterwards as well.  Why he closed this, I don't know

**Comment by kquet** [comment]
@kenberland, it was unclear if @soonahn was joking. I will make efforts in the future to place everything within the translations, of course the Koreans, and French matter.

**Comment by kquet** [comment]
screen shot 2016-01-27 at 10 05 38 pm

**Comment by kenberland** [comment]
How are you going to support right-to-left Arabic script?

**Comment by kquet** [comment]
@kenberland give [this](https://www.youtube.com/watch?v=bNfWJ182Nkc&feature=youtu.be&t=23) a try?

**Comment by kquet** [comment]
screen shot 2016-01-28 at 5 25 00 pm

Stop with the shinnannigganz, pls merge @kenberland @soonahn

**Comment by kquet** [comment]
@kenberland @soonahn pls.

**Comment by kquet** [comment]
This is blocking another story.

**Pull request merged**